src/bscPlugin/validation/ScopeValidator.ts

```diff
@@ -288,18 +293,8 @@ export class ScopeValidator {
         //if this is a `createObject('roSGNode'` call, only support known sg node types
         if (firstParamStringValueLower === 'rosgnode' && isLiteralExpression(call?.args[1])) {
             const componentName: Token = call?.args[1]?.tokens.value;
-            //don't validate any components with a colon in their name (probably component libraries, but regular components can have them too).
```

Comment:
refactored some of this code to make checking specific KINDs of function calls easier - eg. callfunc args are offset by 1 vs. real function call args